addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -204,8 +204,8 @@
                     </div>
                     <!-- We keep "18rem" of space to display the points / nickname.
                     Then, the length of the bar is a percentage of the attendee's score compared to the max_score. -->
-                    <t t-set="width_ratio" t-value="round(round(score['scoring_total']) / round(max_score), 3)"/>
-                    <t t-set="width_ratio_question" t-value="str(round(round(score.get('question_score', 0)) / round(score.get('max_question_score', 1)), 3))"/>
```

Comment:
I think there is a type in "score.get('max_question_score', 3)". Shouldn't it be rather score.get('max_question_score', 1) ?

It was already like that but why round every value instead of rounding the final result ?

Reply:
Indeed, thank you!

> why round every value instead of rounding the final result

I don't know, leaving as is in stable because it was like that and not causing crashes ;)